src/main/java/equipments/technology/AbstractTechnologyEquipment.java

```diff
@@ -0,0 +1,29 @@
+package equipments.technology;
+
+import equipments.IEquipment;
+
+public abstract class AbstractTechnologyEquipment implements IEquipment {
+    public String IP;
+    public String technologyEquipmentType;
```

Comment:
if you have an enum named `technologyEquipmentType`, you can't also have a property name `technologyEquipmentType` that isn't of the same type